src/main/java/com/sleekbyte/tailor/common/ConfigProperties.java

```diff
@@ -5,7 +5,7 @@
 import java.util.Properties;
 
 /**
- * Configuration Properties container.
+ * YamlConfiguration Properties container.
```

Comment:
YamlConfiguration makes it explicitly clear that the class is related to .tailor.yml file.